performance/main.py

```diff
@@ -67,7 +67,7 @@ def main():
 
 
 
-@app.route("/performance/<period>")
+@app.route("/api/v1/performance/<period>")
```

Comment:
this should be squashed with corresponding patch, 

and we should use blueprints, please take a look at other projects, for example health APIs for reference